### ISSUE TITLE
encstatefulxz: corrupted data

### ISSUE THREAD

**Issue by NotDarkn** [FWNavy/CryptoSmite#10]
## Built Using?
EndeavourOS with KDE Plasma 6.0.1

## Describe your Issue?
Upon using the latest current commit of the reenrollment branch (6d15131), it reports "encstatefulxz: corrupted data" when trying to extract to encstatefulxz.
```
Setting up encstateful
Wiping and mounting encstateful
Dropping encstateful key
Extracting bckup to encstatefulxz: corrupted data
tar: Child returned status 1
tar: Error is not recoverable: exiting now
tar -xJf "$BACKUP_PAYLOAD" -C "$ENCSTATEFUL_MNT" --checkpoint=.100 failed with exit code 2.
local host / #
```
## Conclusion
The `cryptosmite.sh` needs to change `tar -xJf` to `tar -xf`, which I fixed in PR #8.

When it does, the `cryptosmite.sh` file works as it should and unenrolls the Chromebook.

**Comment by NotDarkn** [comment]
made pr #8

**Comment by MunyDev** [comment]
Fixed #9 from merge
